### ISSUE TITLE
ascii error when guessing

### ISSUE THREAD

**Issue by duramato** [guessit-io/guessit#492]
```guessit "Re-Animator.1985.INTEGRAL VERSION LIMITED EDITION.1080p.BluRay.REMUX.AVC.DTS-HD MA 5.1-LAZY"
For: Re-Animator.1985.INTEGRAL VERSION LIMITED EDITION.1080p.BluRay.REMUX.AVC.DTS-HD MA 5.1-LAZY
Traceback (most recent call last):
  File "/usr/local/bin/guessit", line 11, in <module>
    load_entry_point('guessit==2.1.4', 'console_scripts', 'guessit')()
  File "/usr/local/lib/python2.7/dist-packages/guessit/__main__.py", line 157, in main
    guess_filename(filename, options)
  File "/usr/local/lib/python2.7/dist-packages/guessit/__main__.py", line 60, in guess_filename
    print('GuessIt found:', json.dumps(guess, cls=GuessitEncoder, indent=4, ensure_ascii=False))
  File "/usr/lib/python2.7/json/__init__.py", line 250, in dumps
    sort_keys=sort_keys, **kw).encode(obj)
  File "/usr/lib/python2.7/json/encoder.py", line 209, in encode
    chunks = list(chunks)
  File "/usr/lib/python2.7/json/encoder.py", line 434, in _iterencode
    for chunk in _iterencode_dict(o, _current_indent_level):
  File "/usr/lib/python2.7/json/encoder.py", line 408, in _iterencode_dict
    for chunk in chunks:
  File "/usr/lib/python2.7/json/encoder.py", line 442, in _iterencode
    o = _default(o)
  File "/usr/local/lib/python2.7/dist-packages/guessit/jsonutils.py", line 30, in default
    return str(o.name)
UnicodeEncodeError: 'ascii' codec can't encode character u'\xc9' in position 1: ordinal not in range(128)
```
```
+-------------------------------------------------------+
+                   GuessIt 2.1.4                       +
+-------------------------------------------------------+
+                   Rebulk 0.9.0                        +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
```

**Comment by shubhank008** [comment]
Same error, filename I am trying to use is "[Doki] Re Zero kara Hajimeru Isekai Seikatsu - 01 1920x1080 Hi10P BD FLAC [7F64383D].mkv"

**Comment by shubhank008** [comment]
@ratoaq2 how do we use the patch/commit ? Its still not available as part of 2.1.4 if trying to upgrade using PIP
